packages/verification/src/rest/v1/verifications/verifications-api.ts

```diff
@@ -254,6 +254,11 @@ export class VerificationsApi extends VerificationDomainApi {
       'Content-Type': 'application/json; charset=UTF-8',
       'Accept': 'application/json',
     };
+    if (data.startVerificationWithSmsRequestBody.smsOptions?.locale !== undefined) {
+      headers['Accept-Language'] = data.startVerificationWithSmsRequestBody.smsOptions?.locale;
```

Comment:
NIT but question mark do not have added value here because checked just above

Reply:
Fixed